evap/contributor/views.py

```diff
@@ -241,7 +243,7 @@ def evaluation_preview(request, evaluation_id):
     ):
         raise PermissionDenied
 
-    return render_vote_page(request, evaluation, preview=True)
+    return render_vote_page(request, evaluation, preview=True, dropout=False)
```

Comment:
This means that we cannot preview the "dropout experience". I think this is fine for this PR, but we should probably later add a way for responsibles to preview what it looks like to drop out @janno42 ?

Reply:
I don't think responsibles need to see this. In practice I expect all dropout questionnaires to be locked so that only staff users are in control of selecting them.